homeassistant/components/roborock/strings.json

```diff
@@ -428,6 +428,24 @@
     "get_maps": {
       "name": "Get maps",
       "description": "Get the map and room information of your device."
+    },
+    "vacuum_goto": {
```

Comment:
I find the name of this action pretty strange. Should it not be "Go to position"?